.env

```diff
@@ -0,0 +1,2 @@
+SQLALCHEMY_DATABASE_URI=sqlite:///postify.db
+SQLALCHEMY_TRACK_MODIFICATIONS=bool(False)
```

Comment:
this is a text file, what I ment by bool is to do in the app.py bool(os.environ['SQLALCHEMY_TRACK_MODIFICATIONS'])